src/rust/graph-descriptions/src/lib.rs

```diff
@@ -62,6 +62,27 @@ impl From<u64> for NodeProperty {
     }
 }
 
+impl From<NodeProperty> for Option<String> {
+    fn from(np: NodeProperty) -> Option<String> {
+        match np.as_str_prop() {
+            Some(str) => Some(String::from(str)),
```

Comment:
Not worth changing, but I think shadowing 'str' is a bit strange, and I'd suggest just using 's' or something. Really minor nit though, nbd.

Also, in this case you can do:

```
np.as_str_prop().map(String::from)
```
and remove the match block entirely.